imgui_widgets.cpp

```diff
@@ -3869,7 +3870,8 @@ static bool InputTextFilterCharacter(unsigned int* p_char, ImGuiInputTextFlags f
     // Custom callback filter
     if (flags & ImGuiInputTextFlags_CallbackCharFilter)
     {
-        ImGuiInputTextCallbackData callback_data;
+        ImGuiContext& g = *GImGui;
+        ImGuiInputTextCallbackData callback_data(&g);
```

Comment:
Seems to be a typo using GImGui here?

Reply:
Yes, it is a type, I will fix it

Reply:
Actually, I don't know why it displays `GImGui` because on the real diff it correctly shows `ctx`.

It is possible you commented the diff of the first commit only, but actually on the diff of the branch there is no problem.

PS: On the first commit `GImGui` still exist and is still used everywhere